packages/scripts/app-build/bumpBuildNumber.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "fs";
+
+const FILE_PATH = "./app.config.js";
+const buildNumber = process.argv[2];
+
+fs.readFile(FILE_PATH, "utf8", (err, data) => {
+  if (err) {
+    console.error("Error reading app.config.js", err);
+    return;
```

Comment:
```suggestion
    process.exit(1);
```
so the ci also fails if this fails